core/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -253,4 +349,58 @@ public Registration registerDispatchInterceptor(MessageDispatchInterceptor<Query
                             .map(queryHandler -> (MessageHandler<? super QueryMessage<?, ?>>) queryHandler)
                             .collect(Collectors.toList());
     }
+
+    private class SimpleQueryUpdateEmitter<U> implements QueryUpdateEmitter<U> {
+
+        private UpdateHandler<?, U> updateHandler;
```

Comment:
Could these global variables be made `private final`?

Reply:
Only `lock` can be final here. Will do that.

Reply:
Thanks!